cfgen/cfgen

```diff
@@ -1074,11 +1075,13 @@ class ConfNode(ToDhall):
     _downlinks = {Downlink('processes', ObjT.process)}
 
     def __init__(self, name: str, nr_cpu: int, memsize_MB: int,
-                 processes: List[Oid], machine_id: Optional[str] = None):
+                 processes: List[Oid], machine_id: Optional[str] = None,
+                 network_ports=None):
```

Comment:
Can you add a type hint for network_ports?

Reply:
Done